### PR TITLE
show report name and CDM version on toolbar

### PR DESCRIPTION
**Pull request by MariaDolotova**
Adding info about report name and cdm version on toolbar

### CODE REVIEW

File: UI/src/app/services/upload.service.ts [hunk 1]
```diff
@@ -36,6 +36,7 @@ export class UploadService {
       );
       this.bridgeService.resetAllMappings();
       this.dataService.prepareTables(res, 'source');
+      this.dataService.saveReportName(files[0].name.substr(0, files[0].name.lastIndexOf('.')), 'report')
```
Comment (etokareva) [inline UI/src/app/services/upload.service.ts:39]: move position of dot into separate variable

Let's use slice. Slice is more universal
substr is supported by historical reasons(also substr is not described in JS specification itself)

File: UI/src/app/components/toolbar/toolbar.component.html [hunk 2]
```diff
@@ -8,6 +8,7 @@
   <button mat-icon-button>
     <mat-icon svgIcon="help"></mat-icon>
   </button>
+  <label class="info-label">{{reportName?reportName:"Report name"}} - {{cdmVersion?"CDM v"+cdmVersion:"CDM version"}}</label>
```
Comment (etokareva) [inline UI/src/app/components/toolbar/toolbar.component.html:11]: Please add spaces around ?, :

File: UI/src/app/components/toolbar/toolbar.component.html [hunk 3]
```diff
@@ -8,6 +8,7 @@
   <button mat-icon-button>
     <mat-icon svgIcon="help"></mat-icon>
   </button>
+  <label class="info-label">{{reportName ? reportName : "Report name"}} - {{cdmVersion ? "CDM v"+cdmVersion : "CDM version"}}</label>
```
Comment (costebano) [inline UI/src/app/components/toolbar/toolbar.component.html:11]: please move calculation of reportName and cdmVersion to .ts file(to ngOnInit)

File: UI/src/app/services/upload.service.ts [hunk 4]
```diff
@@ -14,7 +14,7 @@ export class UploadService {
     private snackbar: MatSnackBar,
     private bridgeService: BridgeService,
     private httpService: HttpService,
-    private dataService: DataService
+    private dataService: DataService,
```
Comment (costebano) [inline UI/src/app/services/upload.service.ts:17]: remove comma

**Pull request merged**